pkg/scheduler/config/config.go

```diff
@@ -1,6 +1,89 @@
 package config
 
+import (
+	"io"
+	"time"
+
+	"code.google.com/p/gcfg"
+)
+
 const (
-	// intended to be mesos FrameworkInfo.Name
-	DefaultInfoName = "Kubernetes"
+	DefaultOfferTTL                           = 5 * time.Second    // duration an offer is viable, prior to being expired
+	DefaultOfferLingerTTL                     = 120  * time.Second // duration an expired offer lingers in history
+	DefaultListenerDelay                      = 1 * time.Second    // duration between offer listener notifications
+	DefaultUpdatesBacklog                     = 2048               // size of the pod updates channel
+	DefaultFrameworkIdRefreshInterval         = 30 * time.Second   // interval we update the frameworkId stored in etcd
+	DefaultInitialImplicitReconciliationDelay = 15 * time.Second   // wait this amount of time after initial registration before attempting implicit reconciliation
+	DefaultExplicitReconciliationMaxBackoff   = 2 * time.Minute    // interval in between internal task status checks/updates
+	DefaultExplicitReconciliationAbortTimeout = 30 * time.Second   // waiting period after attempting to cancel an ongoing reconciliation
+	DefaultInitialPodBackoff                  = 1 * time.Second
+	DefaultMaxPodBackoff                      = 60 * time.Second
+	DefaultHttpHandlerTimeout                 = 10 * time.Second
 )
+
+// Example scheduler configuration file:
+//
+// [scheduler]
+//  info-name        = Kubernetes
+//	offer-ttl        = 5s
+//  offer-linger-ttl = 2m
+
+type ConfigWrapper struct {
+	Scheduler Config
+}
+
+type Config struct {
+	OfferTTL                           WrappedDuration `gcfg:"offer-ttl"`
+	OfferLingerTTL                     WrappedDuration `gcfg:"offer-linger-ttl"`
+	ListenerDelay                      WrappedDuration `gcfg:"listener-delay"`
+	UpdatesBacklog                     int             `gcfg:"updates-backlog"`
+	FrameworkIdRefreshInterval         WrappedDuration `gcfg:"listener-delay"`
+	InitialImplicitReconciliationDelay WrappedDuration `gcfg:"initial-implicit-reconciliation-delay"`
+	ExplicitReconciliationMaxBackoff   WrappedDuration `gcfg:"explicit-reconciliantion-max-backoff"`
+	ExplicitReconciliationAbortTimeout WrappedDuration `gcfg:"explicit-reconciliantion-abort-timeout"`
+	InitialPodBackoff                  WrappedDuration `gcfg:"initial-pod-backoff"`
+	MaxPodBackoff                      WrappedDuration `gcfg:"max-pod-backoff"`
+	HttpHandlerTimeout                 WrappedDuration `gcfg:"http-handler-timeout"`
+}
+
+type WrappedDuration struct {
```

Comment:
I think it's more idiomatic to use an anonymous member here. For an example see how the k8s team wrapped Time: https://github.com/GoogleCloudPlatform/kubernetes/blob/master/pkg/util/time.go

@ConnorDoyle also wants to wrap Duration as part of #210 -- we should probably have a common wrapper that gets reused.